tests/js/environment.js

```diff
@@ -1,23 +1,28 @@
 var fluid = require("infusion");
 
-fluid.defaults("gpii.pouch.tests.environment", {
+fluid.defaults("gpii.test.pouch.environment", {
     gradeNames: ["fluid.test.testEnvironment"],
     port:       6792,
     baseUrl:    "http://localhost:6792/",
     testUrl:    "/sample/",
     events: {
-        constructServer: null,
-        onStarted: null
+        constructFixtures: null,
+        onHarnessReady: null,
+        onFixturesConstructed: {
+            events: {
+                onHarnessReady: "onHarnessReady"
+            }
+        }
```

Comment:
The issue is that you have double-relayed all of the options out from pouch up to harness up to testEnvironment - with the exception of "databases" which is only relayed one step up to harness. To cut down the duplication I suggest you aggregate all of these into an options area named "pouchConfig" and just relay that in bulk to the testEnvironment